test/cbmc/proofs/run-cbmc-proofs.py

```diff
@@ -149,7 +149,7 @@ def run_cmd(cmd, **args):
 
 
 def run_build(litani, jobs):
-    cmd = [str(litani), "run-build"]
+    cmd = [str(litani), "run-build", "--exit-zero-on-fail"]
```

Comment:
Is this a standard option on litani?  Does this mean we have to update every repository based on litani to include this flag in the run script?  Would it not be cleaner to make this is default on litani?

Reply:
> Is this a standard option on litani? 

It is. It seems that I wrote this flag for this exact reason, but didn't get a chance to add it to these run scripts early on.

> Does this mean we have to update every repository based on litani to include this flag in the run script?

Yes

> Would it not be cleaner to make this is default on litani?

This would require a PR to every litani-based repository anyway. I also don't think it's cleaner. Returning zero on failure isn't an intuitive or expected behaviour IMHO.